### ISSUE TITLE
Getting this error again and again in IOS

### ISSUE THREAD

**Issue by GobindaSaha** [Expensify/react-native-share-menu#78]
CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
![Screen Shot 2020-09-09 at 11 53 54 PM](https://user-images.githubusercontent.com/34857385/92638402-d86baa80-f2f7-11ea-801c-990bb630d66d.png)

**Comment by AndyUGA** [comment]
I know it's a late reply but I was getting a similar error and this is what I did to fix it for anyone else running into a similar issue.

1. I recloned the repo. 
2. Make sure change into the example folder of the current repo
3. Run 'yarn install'
4. Then change into the ios folder and run 'pod install'
5. Then open Xcode and open the 'Test.xcworkspace' file located in './example/ios/Test.xcworkspace'
6. Then build the application and it should work.

Hopefully this works for you or anyone else who is running into a similar issue!

**Comment by quriosapien** [comment]
@GobindaSaha
Are you still facing this issue or is it resolved ??
Can we proceed with closing this ticket ?

You can refer this - https://github.com/CocoaPods/CocoaPods/issues/9884#issuecomment-696222756

tldr; Add this to your Podfile to change to iPhoneOS Deployment Target to 9.0
```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '9.0'
    end
  end
end
```

Please execute `pod install` in the `ios` folder of your project after adding this snippet.
